src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java

```
@@ -797,4 +799,70 @@ public void cc_Advisor_selects_the(String statusSelected) {
 
     return requestModifyCaseResponse;
   }
+
+  @And("an empty queue exists for sending SurveyLaunched events")
+  public void an_empty_queue_exists_for_sending_SurveyLaunched_events() throws CTPException {
+    EventType eventType = EventType.valueOf("SURVEY_LAUNCHED");
+    log.info("Creating queue for events of type: '" + eventType + "'");
+    queueName = rabbit.createQueue(eventType);
+    log.info("Flushing queue: '" + queueName + "'");
+    rabbit.flushQueue(queueName);
+  }
+
+  @When("CC Advisor selects the survey launch")
+  public void cc_Advisor_selects_the_launch() {
+    try {
+      log.with(caseId)
+          .info("Now putting a ModifyCaseRequestDTO on the modifyCase endpoint for this case id..");
+      final String response = requestSurveyLaunch();
+      log.info("SURVEY LAUNCH: The response from " + telephoneEndpointUrl.toString());
+      assertNotNull("SURVEY LAUNCH failure", response);
+    } catch (Exception e) {
+      fail("SURVEY launch HAS FAILED - the contact centre does not give a response code of 200");
+    }
+  }
+
+  private String requestSurveyLaunch() {
+    log.with(telephoneEndpointUrl).info("The url for requesting the survey launch");
+
+    String launchSurveyResponseString = null;
+    try {
+      launchSurveyResponseString =
+          getRestTemplate().getForObject(telephoneEndpointUrl, String.class);
+    } catch (Exception ex) {
+      System.out.println(ex.getMessage());
```

Why not just let the test fail?

---

Added
      fail("SURVEY launch HAS FAILED - the contact centre does not give a response code of 200");